mlir/include/mlir/Dialect/EmitC/IR/EmitC.td

```diff
@@ -246,6 +247,81 @@ def EmitC_DivOp : EmitC_BinaryOp<"div", []> {
   let results = (outs FloatIntegerIndexOrOpaqueType);
 }
 
+def EmitC_ForOp : EmitC_Op<"for",
+      [AutomaticAllocationScope, DeclareOpInterfaceMethods<LoopLikeOpInterface,
```

Comment:
Nit: Lets start the declare... on the new line and then indent, just to make it easier to see.